cmd/ci-scheduling-webhook/testing/builds-admission-review-request.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
Everything under `testing/` are hack testing scripts used during development and can be removed. They are too custom to have value to commit them.